.circleci/config.yml

```diff
@@ -644,114 +694,40 @@ jobs:
-      # the src server
-      - image: << pipeline.parameters.container_registry >>/<< pipeline.parameters.gcp_project >>/images/local-testcontainer:<< pipeline.parameters.wandb_server_tag >>
-        auth:
-          username: _json_key
-          password: $GCP_SERVICE_ACCOUNT_JSON_DECODED
-        environment:
-          CI: 1
-          WANDB_ENABLE_TEST_CONTAINER: true
-      # the dst server
-      - image: << pipeline.parameters.container_registry >>/<< pipeline.parameters.gcp_project >>/images/local-testcontainer:<< pipeline.parameters.wandb_server_tag >>
-        auth:
-          username: _json_key
-          password: $GCP_SERVICE_ACCOUNT_JSON_DECODED
-        environment:
-          CI: 1
-          WANDB_ENABLE_TEST_CONTAINER: true
-        name: << parameters.server_name2 >>
-
-    resource_class: xlarge
-    working_directory: /mnt/ramdisk
-    environment:
-      SHARD: << parameters.shard >>
-      WANDB_TEST_SERVER_URL2: << parameters.server_url2 >>
-      WANDB_TEST_LOCAL_BASE_PORT2: << parameters.local_base_port2 >>
-      WANDB_TEST_SERVICES_API_PORT2: << parameters.services_api_port2 >>
-      WANDB_TEST_FIXTURE_SERVICE_PORT2: << parameters.fixture_service_port2 >>
+    executor: << parameters.executor >>
+    environment: { SHARD: << parameters.tests >> }
     steps:
       - checkout
       - run:
           name: Install system deps
           command: |
-            apt-get update && apt-get install -y libsndfile1 ffmpeg
+            apt-get update
+            apt-get install -y libsndfile1 ffmpeg
       - run:
           name: Install Python dependencies
-          command: |
-            pip install -U tox nox pip
+          command: pip install -U tox nox pip
           no_output_timeout: 5m
-      - run:
-          name: Wait for test-container startup
-          command: |
-            while ! curl http://localhost:8080/healthz; do sleep 1; done
-            while ! curl << parameters.server_url2>>:<< parameters.local_base_port2 >>/healthz; do sleep 1; done
-          no_output_timeout: 1m
       - run:
           name: Run tests
           no_output_timeout: 10m
           command: |
             CI_PYTEST_PARALLEL=<< parameters.xdist >> \
-            tox run -v -e <<parameters.toxenv>>  \
-              -- --wandb-server-tag='<< pipeline.parameters.wandb_server_tag >>' \
-              << parameters.tox_args >>
-      - when:
-          condition:
-            not:
-              equal: ["", << parameters.codecov_flags >>]
-          steps:
-            - codecov/upload:
-                { flags: <<parameters.codecov_flags>>, validate: false }
+            tox run -v \
+              -e importer-<<parameters.tests>>-py<<parameters.python_version_major>><<parameters.python_version_minor>>  \
+              -- \
+              tests/pytest_tests/system_tests/test_importers/test_<<parameters.tests>>/ \
+              tests/pytest_tests/unit_tests/test_importers/test_<<parameters.tests>>_utils.py/
```

Comment:
can you just do `tests/pytest_tests/system_tests/test_importers/test_<<parameters.tests>>`?

Reply:
You want to remove the unit tests?

Reply:
oh sorry, i missed that these were unit_tests. why do you need the test container for unit tests? just put it with the other unit tests? (the non-sdk in the unit-tests job for others)

Reply:
although looking at the circleci documentation, don't know if there is an easy way (or a way) to do it.

Reply:
could you please move it to the unit tests job (if it is unit tests, if it is system, so move it to system)?